recipe/meta.yaml

```diff
@@ -9,9 +9,11 @@ source:
   sha256: 77a636bdc08c7668a15951894548c527f0c8c5c2abc86cb850de17551af51e3e
 
 build:
-  number: 2
+  number: 3
   script:
     - export LDFLAGS="-headerpad_max_install_names"  # [osx]
+    - rm -f skimage/run-hessian.py
+    - rm -rf skimage/viewer/tests  # we don't depend on Qt
     - python -m pip install --no-deps .
```

Comment:
What if we trying going back to `python setup.py install --single-version-externally-managed --record record.txt`?

Reply:
@jakirkham can you explain why you think this is failing?